EmailNotification/utils.ts

```diff
@@ -108,7 +108,10 @@ type MessageReducedToHtmlInput = {
 };
 
 export const truncateMarkdown = (plainText: string): string =>
-  plainText.substring(0, MAX_CHARACTER_FOR_BODY_MAIL);
+  // we add "..." only when the message is going to be truncate
+  plainText.length > 134
```

Comment:
why not checking  `>MAX_CHARACTER_FOR_BODY_MAIL`?

Reply:
Absolutely right, did it in [d91490c](https://github.com/pagopa/io-functions-services/pull/280/commits/d91490ca49cf154c2036cdb2a6ad9ada3778d0d4)